aws-observability-terraform/main.auto.tfvars

```diff
@@ -5,3 +5,4 @@ sumologic_access_id       = "<YOUR SUMO ACCESS ID>"    # Please replace <YOUR SU
 sumologic_access_key      = "<YOUR SUMO ACCESS KEY>"   # Please replace <YOUR SUMO ACCESS KEY> (including brackets) with your Sumo Logic Access KEY.
 sumologic_organization_id = "<YOUR SUMO ORG ID>"       # Please replace <YOUR SUMO ORG ID> (including brackets) with your Sumo Logic Organization ID.
 aws_account_alias         = "<YOUR AWS ACCOUNT ALIAS>" # Please replace <YOUR AWS ACCOUNT ALIAS> with an AWS account alias for identification in Sumo Logic Explorer View, metrics and logs.
+sumologic_folder_installation_location = "<YOUR APP FOLDER LOCATION>" # Please replace <YOUR APP FOLDER LOCATION> with your preferred folder installation location, either of Personal or Admin.
```

Comment:
Comment should be
# Please replace <YOUR APP FOLDER LOCATION> with your preferred folder installation location, either of "Personal Folder" or "Admin Recommended Folder". Default is "Personal Folder"